src/NexusMods.Abstractions.GameLocators/Stores/GOG/GOGLocatorResultMetadata.cs

```diff
@@ -16,13 +16,13 @@ public record GOGLocatorResultMetadata : IGameLocatorResultMetadata
     /// <summary>
     /// The build ID of the found game.
     /// </summary>
-    public required string BuildId { get; init; }
+    public required ulong BuildId { get; init; }
```

Comment:
Just to check, there is supposed to be a single build Id per game installation, even if DLCs are installed?

Reply:
Yeah, DLCs are in `installedDLCs`.